docs/aerodromes/classc/Amberley.md

```diff
@@ -26,44 +26,82 @@ AMB ADC owns the Class C airspace **in the AMB CTR** within 10nm of the YAMB ARP
 Aircraft who are not planned via those points or who are negative RNAV may be assigned a RADAR SID or a visual departure. 
 
-#### Coded Departures (Fast Jets)
-Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. BYRON7 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane, avoiding civilian traffic inbound to Brisbane. 
-
-Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.
+### Coded Departures (Fast Jets)
+Visual Departures are commonly requested by high performance military jets and often processed in the form of a coded departure (e.g. BYRON1 departure), which can be found in the AD2 Sups Page for YAMB. These coded departures provide a corridor with altitude constraints to join the Military Training Areas east of Brisbane, avoiding civilian traffic inbound to Brisbane. 
 
 !!! phraseology
-    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.
-    **AMB ADC**: "CRNG21, cleared to YAMB via MOSSI, flight planned route. Runway 15, Northern 1 Departure. Climb to `F190`. Squawk 3601, Departure Frequency 126.20"  
+    CRNG21 plans to enter R637 via the gate MOSSI for Military Training.  
+    **AMB ADC**: "CRNG21, cleared YAMB via MOSSI, flight planned route. Runway 15, Northern 3 Departure. Climb `F190`. Squawk 3601, Departure Frequency 126.20"   
+
+Aircraft will depart the circuit visually and track overhead YAMB to begin tracking for their initial waypoint.
 
 !!! phraseology
-    WOLF03 was assiged the BYRON7 departure with their initial clearance. 
+    WOLF03 was assiged the BYRON1 departure with their initial clearance.  
     **AMB ADC**: "WOLF03, make left turn, reach `F190` by COWIE, cleared for takeoff"  
     **WOLF03**: "Make left turn, reach `F190` by COWIE, cleared for takeoff, WOLF03"  
 
 | Departure | Initial Constraint | Route |
 | --------- | ----------| --------- |
-| Northern 1 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
-| Central 1 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
-| Byron 7 | `F190` at BINUP | COWIE KIWEE ZANET LOTMA SEMAJ |
-
-!!! note
-    Charts for the Coded Departures may be in YAMB AD2 Supplements 6.2.25.11 available here: [RAAF AIP page](https://ais-af.airforce.gov.au/australian-aip){target=new}
+| Northern 3 | `F190` at BINUP | BINUP MURJO BOBED LESKO MOSSI/ADNUK (Transition) |
+| Northern 4 | `F200` at ANKED | ANKED JEZZA BOBED LESKO MOSSI/ADNUK (Transition) |
+| Central 3 | `F190` at BINUP | BINUP MURJO BOBED VIRGE |
+| Central 4 | `F200` at ANKED | ANKED JEZZA BOBED VIRGE |
+| Byron 1 | `F190` at COWIE | COWIE KIWEE ZANET LOTMA SEMAJ |
+| Byron 2 | `F200` at TUGUN | TUGUN LAGOB ZANET LOTMA SEMAJ |
```

Comment:
Do you know if these procedures get updated through successive AIRACs? We won't have any indication that they have without manually checking charts (we don't scrape the RAAF charts)

Reply:
No idea, these name changes were effective from 28th Nov 2024 according to the AD2 SUP.

Reply:
> Do you know if these procedures get updated through successive AIRACs? We won't have any indication that they have without manually checking charts (we don't scrape the RAAF charts)

I thought about this too. Since we can't refer to them as the "Byron SID" or the "Northern SID" (since they're not SIDs), it's a bit awkward. Maybe we can refer to them as the "Byron Coded Departure"? Otherwise I'm just happy to leave the number in there and roll the dice on future AIRAC updates. What do you reckon?

Reply:
I'm happy to roll the dice too, i pretty often take a glance at different AD2 SUPs just to see what's changed. Would be more then happy to create another PR if new updates for the coded departures occur and no one has done so already

Reply:
Yeah great, I'm happy with this. Given that we don't have AIP charts for these procedures, most controllers will directly reference the SOPs (rather than the RAAF TERMA) when providing these instructions, so having the procedure number is beneficial.
@llamavert feel free to submit any AIRAC updates as you find them